app/logic/tasks.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Tasks
+  extend self
+
```

Comment:
I think we can yet keep all logic in the `Competition` module as this logic 100% relates to competition.